### PR TITLE
[StateCodelab] Bump to alpha06

### PR DESCRIPTION
**Pull request by objcode**

### CODE REVIEW

**Review by Kaaveh — APPROVED** [review]

File: StateCodelab/start/src/main/java/com/codelabs/state/todo/Data.kt [hunk 1]
```diff
@@ -23,7 +23,7 @@ import androidx.compose.material.icons.filled.Event
 import androidx.compose.material.icons.filled.PrivacyTip
 import androidx.compose.material.icons.filled.RestoreFromTrash
 import androidx.compose.ui.graphics.vector.VectorAsset
-import java.util.UUID
+import java.util.*
```
Comment (yrezgui) [inline StateCodelab/start/src/main/java/com/codelabs/state/todo/Data.kt:26]: Single import

File: StateCodelab/start/src/main/java/com/codelabs/state/todo/TodoComponents.kt [hunk 2]
```diff
@@ -129,17 +136,14 @@ private fun SelectableIconButton(
...
         MaterialTheme.colors.onSurface.copy(alpha = 0.6f)
     }
-    Button(
+    TextButton(
         onClick = { onIconSelected() },
         shape = CircleShape,
-        backgroundColor = Color.Transparent,
-        border = null,
-        elevation = 0.dp,
+        colors = ButtonConstants.defaultTextButtonColors(),
```
Comment (yrezgui) [inline StateCodelab/start/src/main/java/com/codelabs/state/todo/TodoComponents.kt:142]: It's not needed, default value is already `ButtonConstants.defaultTextButtonColors()`

File: StateCodelab/finished/src/main/java/com/codelabs/state/todo/TodoComponents.kt [hunk 3]
```diff
@@ -129,17 +136,14 @@ private fun SelectableIconButton(
...
         MaterialTheme.colors.onSurface.copy(alpha = 0.6f)
     }
-    Button(
+    TextButton(
         onClick = { onIconSelected() },
         shape = CircleShape,
-        backgroundColor = Color.Transparent,
-        border = null,
-        elevation = 0.dp,
+        colors = ButtonConstants.defaultTextButtonColors(),
```
Comment (yrezgui) [inline StateCodelab/finished/src/main/java/com/codelabs/state/todo/TodoComponents.kt:142]: It's not needed, default value is already `ButtonConstants.defaultTextButtonColors()`

**Review by yrezgui — CHANGES_REQUESTED** [review]
Quick changes and LGTM

**Comment by yrezgui** [comment]
I changed the target branch to `dev_alpha06_main`

File: StateCodelab/start/src/main/java/com/codelabs/state/todo/Data.kt (hunk 1 above)
Comment (objcode) [reply StateCodelab/start/src/main/java/com/codelabs/state/todo/Data.kt:26]: Any way to got AS to stop making this edit? :)

**Comment by chrisbanes** [comment]
Bump (the `dev_alpha06_end` also needs to be updated)

**Review by yrezgui — APPROVED** [review]

**Pull request merged**